### PR TITLE
assertions for readMultisep

### PR DESCRIPTION
**Pull request by grewered**

### CODE REVIEW

**Comment by grewered** [comment]
I addd a check whether the file exists (in the working directory). I hadn't used it before, so maybe it's causing trouble?

**Review by weirichs — COMMENTED** [review]
I think `checkmate::assert_file(file)` is a little bit redundant or superfluous, because if the file doesn't exist, an equivalent error message is given by `readLines`. This line could be deleted. `checkmate::assert_character(sep)` works well.

**Comment by grewered** [comment]
I deleted the assertion for `file`, but it's still not working, though..

File: R/readMultisep.R [hunk 1]
```diff
@@ -1,8 +1,7 @@
 readMultisep <- function(file,sep) {
+  checkmate::assert_character(sep)
```
Comment (sachseka) [inline R/readMultisep.R:2]: please add len=1

**Review by sachseka — APPROVED** [review]
sorry no idea why the codecov check fails; maybe just re-run? or are some tests failing?

File: R/readMultisep.R (hunk 1 above)
Comment (sachseka) [reply R/readMultisep.R]: hmm, i suppose i should write a test for this function -- will put this on my todo

Comment (grewered) [reply R/readMultisep.R]: but shall I merge or just leave this pull request open?

Comment (sachseka) [reply R/readMultisep.R]: yes please merge, i will do this separately

**Pull request merged**